integrations/extensions/starter-kits/language-model-watsonx/README.md

```diff
@@ -0,0 +1,76 @@
+# IBM watsonx language model starter kit
+
+watsonx is our enterprise-ready AI and data platform designed to multiply the impact of AI across your business. It provides an API for interacting with generative language models. You can use this starter kit to connect IBM Watson Assistant to the watsonx API.
+
+This starter kit does not provide any examples of anything that you can _do_ with this extension. It can be used for specific use cases as the generative language model component.
+
+The watsonx specification in the starter kit describes one endpoint, and a few of the most important of the configuration options that this endpoint provides.
+
+| Endpoint   | Description                                                                                                                                                                 |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Generation | Used with watsonx text completion models such as `google/flan-ul2` and `google/flan-t5-xxl`. You provide text as a prompt and it returns the text that follows that prompt. |
+
+## Prerequisites
+
+### Create an API key
+
+1. Log into [watsonx](https://dataplatform.cloud.ibm.com/wx/home?context=wx&apps=cos&nocache=true&onboarding=true&quick_start_target=watsonx) and [generate an API key](https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/ml-authentication.html?context=cpdaas). Save this API key somewhere safe and accessible. The initial version of this starter kit uses temporary access tokens only, so you need to [use your API key to authenticate with an IAM token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey), which you will need to refresh periodically.
```

Comment:
I feel like we probably need a much more detailed description of how to make an API key and a bearer token, but maybe we can put that off until a future release because we're in a hurry now.

Reply:
The information in the links provided should be enough. Any additions here would repeat that content.

Reply:
OK, cool